src/components/flowchart/flowchart.js

```diff
@@ -711,6 +717,17 @@ export class FlowChart extends Component {
   /**
    * Render React elements
    */
+  renderNotification() {
```

Comment:
hey @SajidAlamQB perhaps `SlicedPipelineActionBar` can be reused here, rather you have to make a new component. You might need to change a bit logic inside `SlicedPipelineActionBar` but it has the styling and theme already set up in there